ext/cfx-ui/src/cfx/common/services/activity/ActiveActivityPubFeed.ts

```diff
@@ -171,7 +171,7 @@ export class ActiveActivityPubFeed {
       },
     });
 
-    if (!isObject<IActivityPubAccount>(account) || account.type !== 'Person') {
+    if (!isObject<IActivityPubAccount>(account) || !(account.type === 'Person' || account.type === 'Service')) {
```

Comment:
Having one negation for the whole condition would make it more readable in my opinion:
```ts
if (!(isObject<IActivityPubAccount>(account) && (account.type === 'Person' || account.type === 'Service'))) {
    throw new Error(`Unknown or invalid account for pub ${this.id}`);
}
```